src/libs/auth.js

```diff
@@ -221,6 +225,7 @@ export const processUser = (user, isSignInEvent) => {
       cookiesAccepted: isSignedIn ? state.cookiesAccepted || getLocalPrefForUserId(userId, cookiesAcceptedKey) : undefined,
       isTimeoutEnabled: isSignedIn ? state.isTimeoutEnabled : undefined,
       hasGcpBillingScopeThroughB2C: isSignedIn ? state.hasGcpBillingScopeThroughB2C : undefined,
+      seenAzurePreviewScreen: isSignedIn ? state.seenAzurePreviewScreen : undefined,
```

Comment:
This tracks whether a user has seen the preview screen, so we don't keep showing it.

TODO: right now this is reset if you refresh the page; product/UX instead want this tied to login/logout. Investigate using browser local storage for this instead.